application/forms/DimensionsForm.php

```diff
@@ -7,11 +7,22 @@
 use Icinga\Module\Cube\Cube;
 use Icinga\Module\Cube\Dimension;
 use Icinga\Module\Cube\DimensionParams;
-use Icinga\Web\Form;
 use Icinga\Web\Notification;
+use ipl\Html\Form;
+use ipl\Html\Html;
+use ipl\Web\Common\FormUid;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
 
 class DimensionsForm extends Form
 {
+    use FormUid;
```

Comment:
Also use the `Translation` trait. There will be change soon in ipl-html which causes it to not being part of `ipl\Html\Form` anymore.